### ISSUE TITLE
Can't play osu with new input settings...

### ISSUE THREAD

**Issue by axrmonkey** [ppy/osu#12120]
**Describe the bug:** Raw input is gone, can't set sensitivity without "High precision mouse" and this new "Tablet Feature" is not working and keeps giving me an error "Device: Failed to set tablet feature: 02-02". All I used to do was set the sensitivity high while also using raw input while my mouse was bound to one monitor. Now when I set it to one monitor, "High precision mouse", and with my sensitivity up it just keeps my cursor stuck to one side of the screen. If I set it to my right monitor it would be stuck to the right and left if set to my left monitor. I've tried messing with settings on my tablet but to no avail it is unplayable. All these issues above involve osu being in fullscreen but when trying to mess with the sensitivity in windowed or borderless the sensitivity is unchanged.

**Screenshots or videos showing encountered issue:** https://www.youtube.com/watch?v=OjR0pYeNhPo

**osu!lazer version:** 2021.320.0

**Logs:**
[network.log](https://github.com/ppy/osu/files/6176815/network.log)
[performance.log](https://github.com/ppy/osu/files/6176816/performance.log)
[runtime.log](https://github.com/ppy/osu/files/6176817/runtime.log)
[updater.log](https://github.com/ppy/osu/files/6176818/updater.log)
[database.log](https://github.com/ppy/osu/files/6176819/database.log)

**Comment by peppy** [comment]
I'm confused. are you reporting an issue with mouse or tablet? Could you please reword your report to focus on one or the other?

Also can you try running windowed to see if that helps?

**Comment by axrmonkey** [comment]
Sorry, it is a tablet issue but I used to use the "raw input" option from before while increasing the sensitivity in order to play. I have also tried windowed but it gives the same effect as borderless where the sensitivity does not change no matter what the sensitivity is set to.

**Comment by bdach** [comment]
I'm not sure how increased sensitivity on a tablet device (one which does absolute input) would work. Would you mind explaining? Does that mean you effectively use a working area that is larger than your screen, and you can't physically reach some areas of the screen using the tablet?

Also, with regards to the error, that is something to report to OTD directly. Before trying that, though, did you try to disable the Wacom driver?

**Comment by axrmonkey** [comment]
Well when I use my preferred settings it lets me use a very small portion of my tablet (about half an inch) than my whole tablet. I have tried to recreate these settings by limiting the mapping on the Wacom settings but it does make the area as small as im used to and also feels a bit warped. I have tried to disable the driver but then it does not read my pen inputs.

**Comment by bdach** [comment]
I see. Your setup *should* in the long term be reproducible through the in-game settings, but simply by setting a smaller area, something like so:

![2021-03-21-102520_546x369_scrot](https://user-images.githubusercontent.com/20418176/111899907-cb124400-8a2f-11eb-8043-dcdba51290d2.png)

So I suppose the larger issue is that OpenTabletDriver refuses to cooperate at all, which renders this impossible. Then again, I'm not sure we can help out with that here.

**Comment by axrmonkey** [comment]
Yeah I've tried messing with that new setting but I see no change the same way I see no change when I increase the sensitivity while having it set to windowed or borderless.

**Comment by bdach** [comment]
That will presumably be due to [this issue](https://github.com/ppy/osu-framework/issues/4294).

Not sure whether there's much reason to keep this issue open at this stage, since the problems mentioned seem to mostly be either out of our scope, or already tracked elsewhere.

**Comment by axrmonkey** [comment]
Well so far I can play again with OTD I just wish I was able to play as I was before but oh well. Thanks bdach 👍
